nessus-agent/elx/configure.sls

```diff
@@ -0,0 +1,60 @@
+# This salt state installs the Nessus Agent using the provided
+# URL to the rpm source file.  Once installed, the agent will be
+# configured and linked to the Nessus server using the provided
+# parameters (Nessus 64 hexadecimal-digit key, Nessus server FQDN and
+# port, Nessus Agent Group Name).  The final step is to start tHe
+# Nessus Agent service.
+#################################################################
```

Comment:
The description needs to be updated to describe the new configure process.

Reply:
Updated (for the vanishingly-small number of users likely to read them 😛).